backend/simple_unified_startup.py

```
@@ -18,37 +18,49 @@ async def initialize_jyotiflow_simple():
+            elapsed_time = time.time() - start_time
+            logger.info(f"✅ JyotiFlow.ai initialized successfully in {elapsed_time:.2f} seconds")
+            logger.info("🎯 Clean shared pool architecture ready")
+            
+            return db_pool
+            
+        except (asyncpg.exceptions.PostgresError, OSError, ConnectionError, TimeoutError) as e:
+            elapsed_time = time.time() - start_time
+            logger.warning(f"⚠️ Attempt {attempt + 1} failed after {elapsed_time:.2f}s: {e}")
+            if attempt < max_retries - 1:
+                logger.info(f"Retrying in {retry_delay} seconds...")
+                await asyncio.sleep(retry_delay)
```

Ruff (0.12.2)

Undefined name `asyncio`

(F821)